src/libraries/Common/src/Interop/OSX/Interop.Libraries.cs

```diff
@@ -8,13 +8,12 @@ internal static partial class Libraries
         internal const string CoreFoundationLibrary = "/System/Library/Frameworks/CoreFoundation.framework/CoreFoundation";
         internal const string CoreServicesLibrary = "/System/Library/Frameworks/CoreServices.framework/CoreServices";
         internal const string CFNetworkLibrary = "/System/Library/Frameworks/CFNetwork.framework/CFNetwork";
-        internal const string libobjc = "/usr/lib/libobjc.dylib";
-        internal const string libproc = "/usr/lib/libproc.dylib";
+        internal const string @libobjc = "/usr/lib/libobjc.dylib";
+        internal const string @libproc = "/usr/lib/libproc.dylib";
```

Comment:
Does the C# compiler warn about these?  I thought it only warned about lower-cased type names, not consts in general.

Reply:
I did a search/replace to do this change and might have been a little overzealous. I'll go back and clean it up.